daru.gemspec

```diff
@@ -69,7 +69,7 @@ EOF
   spec.add_development_dependency 'dbi'
   spec.add_development_dependency 'activerecord', '~> 4.0'
   spec.add_development_dependency 'sqlite3'
-  spec.add_development_dependency 'rubocop', '>= 0.40.0'
+  spec.add_development_dependency 'rubocop', '~> 0.49.0'
```

Comment:
Not really strong (in my own gems I use both, depending on current mood). But exposing development dependencies in `.gemspec` has advantage of "visibility", for example on [rubygems](https://rubygems.org/gems/daru). Also, there are third-party services like Gemnasium ([example](https://gemnasium.com/github.com/molybdenum-99/infoboxer)), which allow to control & notify on dependent gem updates—I believe they use `.gemspec` too.

`Gemfile` could be preferred due to shorter syntax, and better granulated groups.

So, probably the matter of taste and circumstances.